rt-forecast/update-format-rt.R

```diff
@@ -68,11 +77,29 @@ region_forecasts <- region_forecasts %>%
 
 region_forecasts$forecast_date <- forecast_date
 
-# Save forecast -----------------------------------------------------------
+# Save submission
 # Dated
 readr::write_csv(region_forecasts,
                  paste0("rt-forecast/submission-files/dated/", forecast_date, "-rt-forecast-submission.csv"))
-#Latest
+# Latest
 readr::write_csv(region_forecasts,
                  paste0("rt-forecast/submission-files/latest-rt-forecast-submission.csv"))
 
+
```

Comment:
Split out code for getting samples from that for submission files to make the code a little clearer